internal/io.h

```diff
@@ -13,11 +13,23 @@
 #define HAVE_RB_IO_T
 struct rb_io;
 
-#include "ruby/io.h"            /* for rb_io_t */
+#include "ruby/io.h"
+
+#include "vm_core.h"
```

Comment:
maybe it is for `rb_execution_context_t`, but we can use `struct rb_execution_context_struct *` and we don't need to include `vm_core.h`.

Reply:
Agreed.